src/js/player.js

```diff
@@ -4200,6 +4200,7 @@ class Player extends Component {
       return;
     }
 
+    // error should contain eventtype, and be passed in
```

Comment:
you mean errorType?

Reply:
Yup, this is a typo. This comment will be removed, and was just meant to show the process of how the `errorType` should be passed through.